base_geoengine/static/src/js/geoengine_view.js

```diff
@@ -578,9 +578,11 @@ openerp.base_geoengine = function(openerp) {
             var vl = new OpenLayers.Layer.Vector(self.name, {
 		styleMap: new OpenLayers.StyleMap({
 		    'default': new OpenLayers.Style({
-			fillColor: '#ee9900',
+			fillColor: '#ff0000',
+//			fillColor: '#ee9900', 
```

Comment:
To define if this color change is specific or not. We definitely don't want commented line there thought